remote-wallet/src/remote_wallet.rs

```diff
@@ -93,15 +120,20 @@ pub struct RemoteWalletManager {
     #[cfg(feature = "hidapi")]
     usb: Arc<Mutex<hidapi::HidApi>>,
     devices: RwLock<Vec<Device>>,
+    trezor_client: Option<Rc<RefCell<Trezor>>>,
```

Comment:
I think we probably want to support multiple trezor devices, don't we? I imagine users may want to use more than one for a transaction: as an account authority vs fee-payer, for instance. Certainly we see that use-case with Ledgers commonly.

It looks like `trezor-client` should make this relatively straightforward. Then I think it would probably make sense to roll trezor device recognition into `update_devices()`

Reply:
Ok, this can be done. I'm going to link some references for the sake of auditing.

https://github.com/trezor/trezor-firmware/blob/main/rust/trezor-client/src/lib.rs#L100

https://github.com/trezor/trezor-firmware/blob/main/rust/trezor-client/src/transport/webusb.rs#L81

https://github.com/trezor/trezor-firmware/blob/main/rust/trezor-client/src/transport/udp.rs#L99